auditlog/admin.py

```diff
@@ -40,6 +40,11 @@ def has_change_permission(self, request, obj=None):
         return False
 
     def has_delete_permission(self, request, obj=None):
+        if request.resolver_match.url_name not in [
+            pattern.name for pattern in self.urls if pattern.name
+        ]:
```

Comment:
@aleh-rymasheuski Could you please explain more about what is happening here?

Also, I think you can move the `[pattern.name for pattern in self.urls if pattern.name]` out of method and define a class attr to don't repeat the loop in each call.